kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -361,10 +375,8 @@
       },
       toggleTopicInWorkingResources(isChecked) {
         if (isChecked) {
-          this.isSelectAllChecked = true;
           this.addToWorkingResourcePool(this.contentList);
         } else {
-          this.isSelectAllChecked = false;
```

Comment:
Removing the setting of `isSelectAllChecked` here is good as it's a computed property.

In my note on that `isSelectAllChecked` property I mentioned that the side-effects of manipulating the working resource pool should probably be moved here.

In the `else` block here, however, rather than `resetWorkingResourcePool` this should instead remove all items in the content list from the working resource pool. `resetWorkingResourcePool` just undoes all changes the user has made since opening the panel, which is not desired here.

In this case, you'd need to do a `this.contentList.forEach(content => this.removeFromWorkingResourcePool(content))` (or something along those lines -- you'll probably also want to be sure to filter out Topics as we aren't permitting full topic selection until #11790 is implemented)